.classpath

```diff
@@ -34,7 +34,6 @@
 	<classpathentry kind="lib" path="/usr/share/java/junit.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/ldapjdk.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/jaxb-api.jar"/>
-	<classpathentry kind="lib" path="/usr/share/java/jaxb/jaxb-impl.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/jakarta-activation/jakarta.activation.jar"/>
```

Comment:
If you're updating classpath for yourself, above probably applies as well. I don't think this package exists on older (F31, F32) Fedoras.

Reply:
I'm trying to find the common denominator so the `.classpath` will work on most develpment platforms if possible (i.e. F31-F33). IIUC `jaxb-impl.jar` is not required on any platform, and also not available on F31, so it can be removed. `jakarta.activation.jar` is not required on F31, but is it required on F33? It's available on F31 so we can leave it here.

Reply:
Interesting, What package provides it on F31? 0.o

Reply:
It's `jakarta-activation-1.2.1-5.fc31.noarch`.